validate.go

```diff
@@ -29,10 +29,10 @@ func adjustResourceRequest(container *corev1.Container, resourceName string, res
 // Return `true` when the container has been mutated
 func validateAndAdjustContainerResourceRequests(container *corev1.Container, settings *Settings) bool {
 	mutated := false
-	if settings.Memory != nil {
+	if !settings.Memory.IgnoreValues && settings.Memory != nil {
```

Comment:
I don't think this is covering what is being said in this README section:

```
If you would only like to enforce that requests and limits are set (ie, you do not care
what they are set to, just that they have been set), you can set `ignoreValues` to `true`.
This will skip the enforcement of specific values and only enforce that requests and
limits are set. Here is an example of how to configure this:
```

For example, consider this unit test:

```go
		{"container missing memory requests values and ignore memory values", corev1.Container{
			Image: "image1:latest",
			Resources: &corev1.ResourceRequirements{
				Limits: map[string]*apimachinery_pkg_api_resource.Quantity{
					"cpu": &oneCoreCpuQuantity,
					"memory": &oneGiMemoryQuantity,
				},
				Requests: map[string]*apimachinery_pkg_api_resource.Quantity{
					"cpu": &oneCoreCpuQuantity,
				},
			},
		}, Settings{
			Cpu: &ResourceConfiguration{
				DefaultLimit:   oneCore,
				DefaultRequest: oneCore,
				MaxLimit:       oneCore,
			},
			Memory: &ResourceConfiguration{
				IgnoreValues: true,
			},
		}, &corev1.ResourceRequirements{
			Limits: map[string]*apimachinery_pkg_api_resource.Quantity{
				"cpu": &oneCoreCpuQuantity,
				"memory": &oneGiMemoryQuantity,
			},
			Requests: map[string]*apimachinery_pkg_api_resource.Quantity{
				"cpu": &oneCoreCpuQuantity,
			},
		}, false, "missing memory resource request"},
// this can be added in the unit tests file. ;-)
```

The policy will not return an error saing: "hey! You need to add the memory in the requested resources".